service/messages/messages.json

```diff
@@ -32,7 +33,13 @@
         "whats your work",
         "what is your work",
         "your work",
-        "work"
+        "work",
+        "I need your help",
+	    "I need your help. Please help me!",
+        "How you work?",
+        "How you work? Please guide me!",
+        "How you work? Let me know!"
```

Comment:
please remove all kinds of symbols. keep it just **plain text**!
the process uses filters, so all kinds of special characters would be removed, so these strings might never find a match with the query.
also, keep everything in ***small-case*** of ***messages.json***. The grammatical punctuations could be maintained only inside the ***messages_replies.json***  file since the replies are directly sent from the respective block